<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoldierApp - game</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>800.000000000000000</width>
       <height>600.000000000000000</height>
      </rectf>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="soldiersManagementLayout">
     <item>
      <widget class="QTableView" name="soldiersTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="orderLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="shootButton">
         <property name="text">
          <string>Shoot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="grenadeButton">
         <property name="text">
          <string>Throw grenade</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="saluteButton">
         <property name="text">
          <string>Salute</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="jumpButton">
         <property name="text">
          <string>Jump</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="moveUpButton">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="moveDownButton">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="orderLowerRankCheckbox">
         <property name="text">
          <string>Order selected with lower rank</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="orderBiggestRankCheckbox">
         <property name="text">
          <string>Order selected with biggest rank</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
